src/components/Card/Card.scss

```diff
@@ -0,0 +1,87 @@
+@import '../src/prepare-styles/colors.scss';
+@import '../src/prepare-styles/reset.scss';
+@import '../src/prepare-styles/typography.scss';
+
+.card_main-container{
+  box-sizing: border-box;
+  padding: 32px;
+  display: flex;
+  flex-direction: column;
+  // height: 439px;
+  // width: 221px;
+  width: min-content;
+  border: 1px solid $grayElements;
+}
+
+.card_container{
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  align-items: center;
+  gap: 8px;
+
+  &-img{
+    width: 148px;
+  }
+
+  &-title{
+    padding-top: 16px;
+  }
+
+  &-info{
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    align-items: start;
+    position: relative;
+
+    &-prices{
+      display: flex;
+      flex-direction: row;
+      gap: 8px;
+      padding-top: 8px;
+
+      ::after{
+        position: absolute;
+        content: '';
+        display: block;
+        bottom: -8px;
+        background-color:  $grayElements;
+        height: 1px;
+        width: 75%;
```

Comment:
i don't think it's a good idea to lock width here at 75%, but it'll do for now. add a comment here so someone would check it later. also, this line is not exactly 75% 🤓 

![image](https://github.com/fs-mar24-kyrylos-witnesses/phone-catalog/assets/100777995/c694ad70-981d-4a97-bf0e-d6669791fc3a)

208 is line width and 272 is card width